hadml/datamodules/components/herwig.py

```diff
@@ -25,8 +25,8 @@
 
 from torch_geometric.data import Data
 from torch_geometric.data import InMemoryDataset
-
 from torch.utils.data import Dataset as TorchDataset
+from torch.utils.data import Dataset
 
 pid_map_fname = "pids_to_ix.pkl"
```

Comment:
This is old code, but now with the fixed typo it needs a fix too ("pids_to_idx.pkl")